docs/docs/examples/database/AMAZON_RDS.md

```diff
@@ -7,10 +7,16 @@ This example provides instructions for creating an Amazon [RDS DB instance](http
 ## Database creation
 There are two steps for creating the database:
 
+1. [Initialize database server](#1-initialize-database-server)
+2. [Initialize database & user](#2-create-database--user)
```

Comment:
/nit `database and user`

The benefit is that the anchor will include `and` instead of a double dash.